add-on/src/popup/browser-action/store.js

```diff
@@ -205,6 +207,7 @@ export default (state, emitter) => {
     const prev = state.active
     state.active = !prev
     if (!state.active) {
+      endSession()
```

Comment:
if we're only calling startSession and endSession inside this DOMContentLoaded for browser-action now, then I don't think we're going to be tracking sessions properly anymore. Is that blocked by https://github.com/ipfs-shipyard/ignite-metrics/issues/103 ?